akka-stream/src/main/scala/akka/stream/scaladsl/Flow.scala

```diff
@@ -1013,11 +1012,7 @@ trait FlowOps[+Out, +Mat] {
    * '''Cancels when''' downstream cancels
    *
    */
-  def flatten[U](strategy: FlattenStrategy[Out, U]): Repr[U, Mat] = strategy match {
-    case scaladsl.FlattenStrategy.Concat | javadsl.FlattenStrategy.Concat ⇒ deprecatedAndThen(ConcatAll())
-    case _ ⇒
-      throw new IllegalArgumentException(s"Unsupported flattening strategy [${strategy.getClass.getName}]")
-  }
+  def flattenConcat[U]()(implicit ev: Out <:< Source[U, _]): Repr[U, Mat] = deprecatedAndThen(ConcatAll())
```

Comment:
<a href="https://imgflip.com/i/tjomq"><img src="https://i.imgflip.com/tjomq.jpg" title="made at imgflip.com"/></a>

Good to stay consistent between DSLs :+1: